src/test/bufferlist.cc

```diff
@@ -1412,7 +1412,8 @@ TEST(BufferList, operator_equal) {
   //
   // list& operator= (list&& other)
   //
-  bufferlist move = std::move(bl);
+  bufferlist move;
+  move = std::move(bl);
```

Comment:
> It should use list::operator= move assignment instead of the move construction function.

why?

Reply:
operator_equal is used to test the move assignment operator instead of testing move constructor.

Reply:
oh, i see! could you revise the commit message a little bit, like:

> bufferlist: test move assignment operator not move constructor
>
> `bufferlist move = std::move(bl)` exercises the move constructor. but the 
> purpose of this test is to verify the functionality of move assignment operator.
> so in this change, construct "move" first, and then assign rvalue reference to it.

?

Reply:
I'll refine the commit message.